eos/utils/options.cc

```diff
@@ -386,4 +387,75 @@ namespace eos
     {
         return _value;
     }
+
+    IsospinOption::IsospinOption(const Options & options, const std::vector<OptionSpecification> & specifications, const std::string & key) :
+        SpecifiedOption(options, specifications, key)
+    {
+    }
+
+    IsospinOption::~IsospinOption() = default;
+
+    uint8_t
+    IsospinOption::value() const
+    {
+        static const std::map<std::string, uint8_t> map
+        {
+            { "0",          1     },
+            { "1",          2     },
+            { "0 & 1",      3     },
+            { "2",          4     },
+            { "0 & 2",      5     },
+            { "1 & 2",      6     },
+            { "0 & 1 & 2",  7     },
+            { "1/2",        8     },
+            { "3/2",        16    },
+            { "1/2 & 3/2",  24    }
+        };
+
+        const auto i = map.find(_value);
+        if (map.cend() == i)
+            throw InternalError("Invalid combination of isospin '" + _value + "' encountered in IsospinOption::value()");
+
+        return i->second;
+    }
+
+    const std::string &
+    IsospinOption::str() const
+    {
+        return _value;
+    }
+
+    PartialWaveOption::PartialWaveOption(const Options & options, const std::vector<OptionSpecification> & specifications, const std::string & key) :
+        SpecifiedOption(options, specifications, key)
+    {
+    }
+
+    PartialWaveOption::~PartialWaveOption() = default;
+
+    uint8_t
+    PartialWaveOption::value() const
+    {
+        static const std::map<std::string, uint8_t> map
+        {
+            { "S",          1     },
+            { "P",          2     },
+            { "S & P",      3     },
```

Comment:
Ditto here. ``enum class PartialWave : uint8_t``, with similar operators as suggested for ``Isospin``?

Reply:
cf #1001